### DISCUSSION TITLE
loading does not show

### DISCUSSION THREAD

**Discussion by karamyan** [filamentphp/filament#11473] [Help]
### Package

Table builder

### Package Version

3.2

### How can we help you?

When you have a table with data and pagination and you click on the pagination, the loading does not show

**Comment by leandrocfe** [comment]
Where would you like to show it?

**Reply by karamyan94** [reply]
In place of table rows  before receiving new data
Оr at least in pagination buttons but it doesn't work either

**Reply by leandrocfe** [reply]
Maybe a "loading" in the table header?

```php
$table
    ->header(view('custom-table-header'))
```

```html
<!-- resources/views/custom-table-header.blade.php -->
<div wire:loading class="p-2">
    <p>Loading...</p>
</div>

```

**Reply by karamyan94** [reply]
Thanks for the answer.
Can I do the same thing not in the header but in the body of the table or in the footer, because I have pagination from the bottom and when I click the header it’s not even visible as a whole

**Reply by karamyan94** [reply]
@leandrocfe  you don`t have any solution ? )
